routes/api/contacts.js

```diff
@@ -120,7 +145,48 @@ router.put("/:contactId", async (req, res, next) => {
         }
     } catch (err) {
         console.log(err.message);
+        res.status(500).json({
+            status: "failure",
+            code: 500,
+            message: err.message,
+        });
     }
 });
 
+router.patch("/:contactId/favorite", async (req, res, next) => {
+    try {
+        const body = {
+            favorite: req.query.favorite,
+        };
+        const result = await updateStatusContact(req.params.contactId, body);
+        if (result && result.status !== 400 && result !== 400) {
+            res.json({
+                status: "success",
+                code: 200,
+                data: result,
+            });
+            return;
```

Comment:
```suggestion
            return res.json({
                status: "success",
                code: 200,
                data: result,
            });
```
Ładniej wygląda taki sposób pisania.